doc/lms.mysql

```diff
@@ -2919,4 +2919,4 @@ INSERT INTO netdevicemodels (name, alternative_name, netdeviceproducerid) VALUES
 ('XR7', 'XR7 MINI PCI PCBA', 2),
 ('XR9', 'MINI PCI 600MW 900MHZ', 2);
 
-INSERT INTO dbinfo (keytype, keyvalue) VALUES ('dbversion', '2017071900');
+INSERT INTO dbinfo (keytype, keyvalue) VALUES ('dbversion', '2017072100');
```

Comment:
To jak podbijamy schemat bazy danych to może i tu dodać wstawienie ustawień konfiguracyjnych?